style.css

```diff
@@ -1 +0,0 @@
-/* TODO: insert default styles of your component here for easy access */
```

Comment:
@Bamieh no CI integration yet but i think the build will fail as you forgot to remove the reference to `style.css` in code, it is loaded inside `doc/index.js`